environments/icds-cas/public.yml

```diff
@@ -251,6 +251,18 @@ localsettings:
   ENTERPRISE_MODE: True
   ENABLE_DRACONIAN_SECURITY_FEATURES: yes
   ICDS_DOMAIN: "{{ localsettings_private.ICDS_DOMAIN }}"
-  UCR_COMPARISONS: {}
+  UCR_COMPARISONS:
+    'static-mpr_10a_person_cases': 'static-mpr_10a_person_cases-optimized'
```

Comment:
oof. didn't get this out in time to stick around for a test, and this is also the incorrect format. should have domain in there like https://github.com/dimagi/commcare-cloud/pull/2348/